hs_core/views/__init__.py

```diff
@@ -696,7 +696,7 @@ class AddUserForm(forms.Form):
                                       widget=autocomplete_light.ChoiceWidget("UserAutocomplete"))
 
     class AddGroupForm(forms.Form):
-        group = forms.ModelChoiceField(Group.objects.filter(gaccess__active=True).all(),
+        group = forms.ModelChoiceField(Group.objects.filter(gaccess__active=True).exclude(name='Hydroshare Author').all(),
```

Comment:
I think there shouldn't be a gaccess record for "HydroShare Author".... That would fix this system wide.